datahub-web-react/src/components/browse/BrowseTypesPage.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import 'antd/dist/antd.css';
+import { Link } from 'react-router-dom';
+import { Col, Row, Card } from 'antd';
+import { Content } from 'antd/lib/layout/layout';
+import { BrowseCfg } from '../../conf';
+import { SearchablePage } from '../search/SearchablePage';
+import { toCollectionName, toPathName } from '../shared/EntityTypeUtil';
+import { PageRoutes } from '../../conf/Global';
+import '../../App.css';
+
+const { BROWSABLE_ENTITY_TYPES } = BrowseCfg;
+
+export const BrowseTypesPage = () => {
+    return (
+        <SearchablePage>
+            <Content style={{ backgroundColor: 'white', padding: '25px 100px' }}>
+                <h1 className="ant-typography">Browse</h1>
+                <Row gutter={[16, 16]}>
```

Comment:
Our next initiative should be to cleanse the inline styling :)

Reply:
yeah, that'd be good to get ahead of 👍